framework/CMakeLists.txt

```diff
@@ -486,6 +486,10 @@ if(${VKB_VALIDATION_LAYERS_GPU_ASSISTED})
     endif()
 endif()
 
+if(${VKB_VALIDATION_LAYERS_BEST_PRACTICES})
```

Comment:
Does it make sense to enforce validation layers to on a they are needed so seeing this also sets validation layers?